### DISCUSSION TITLE
Adding official trails to the maps

### DISCUSSION THREAD

**Discussion by yasio90** [organicmaps/organicmaps#4262] [Ideas]
I would find very helpful if one could see (possibly as an additional layer) official trails marked with their corresponding colors, like in the screen shot below taken from mapy.cz.

![image](https://user-images.githubusercontent.com/8491665/211942556-2dddb45d-a78b-47ba-a4b4-dc56d1452104.png)

**Comment by biodranik** [comment]
It is planned for the "outdoor" or "hiking" map style. I think it was mentioned already in some issues.

**Reply by yasio90** [reply]
Cool! Happy to hear.
